### PR TITLE
Create namespace-registry.md

### PR DESCRIPTION
**Pull request by davaya**
Preface for the registry document, or section in the architecture document that references the registry.

### CODE REVIEW

**Review by Vasileios-Mavroeidis — COMMENTED** [review]
The namespace structure here is a bit "different" than what we already have in the specifications.

http://oasis-open.org/openc2/ap-slpf/v1.0
VS.
oasis-open.org/openc2/v1.0/ap-slpf. (in the SLPF spec) - we also have missed including "http://"

**Comment by OASIS-OP-Admin** [comment]
Vasileiios, the second one is an error. The first one follows the Naming Directives rules for constructing the URLs for the spec: oasis-open.org/<tc-abbrev>/<spec-abbrev>/<version-#>/

File: namespace-registry.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+* **NSID**: a short abbreviation for a Namespace used as a prefix in an imported type: "ex"
+* **Field Name**: may be serialized as a JSON object property whose value is an imported type: "email"
+
+NSID and Field Name are both defined by the importing specification; neither are registered here.
+Type Names (including NSID prefixes) never appear in JSON data, so Namespaces and NSIDs are never
+needed or used in JSON data except within a schema document.
+
+This approach uses a resolver to look up all namespaced type definitions from their defining specifications
+and incorporate them into a single schema. Authors can manually copy and paste definitions
+into a monolithic specification, but namespace resolution automates that process, eliminating redudundancy
```
Comment (dlemire60) [inline namespace-registry.md:44]: typo:  redudandancy --> redundancy

File: namespace-registry.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+A namespace URI is only an identifier. For syntactic reasons it must have a scheme (http) but it
+is not a network-accessible resource. 
+Referenced specifications do not need to be available online and implementations are not required to do
+namespace resolution at runtime, although dynamic namespace resolution may be appropriate for some use cases.
+URLs for online schemas should be derived from the namespace using scheme "https", filename "schema", and
+the applicable file extension: ".jadn" for the abstract schema, and ".json", ".xsd", ".cddl", ".proto", etc.
+for corresponding concrete schemas.
+
+## Registration Process
+OpenC2 TC Documentation Norms suggests naming conventions for TC work products.
```
Comment (dlemire60) [inline namespace-registry.md:56]: Suggest adding link to Documentation Norms, or directly to the naming conventions section of that document.

**Review by TobyConsidine — APPROVED** [review]

**Pull request merged**